### DISCUSSION TITLE
Real world usecase

### DISCUSSION THREAD

**Discussion by TimGeyssens** [TimGeyssens/Umbraco-v8-Community-Website-StarterKit#16] [General]
Might be nice that we build a real world use case, currently thinking a simple real estate agent site + app

**Comment by TimGeyssens** [comment]
Could be an updated version of https://our.umbraco.com/packages/starter-kits/razor-estate-agents-starter-site

**Comment by mzajkowski** [comment]
Hey there, finally got some time to participate there.

I was thinking about a "multi-purpose" starterkit and use-cases as an examples / versions of the site structures. Is it what you mean? Or you're more thinking about StarterKit per Use-Case?

I'm always referring to themes here: https://themeforest.net/search/multipurpose. Ideally I think the way how Frontend and Backend should be build there is to enable us to present what we need using the ways and components we have there.

Of course - there is a new DANGER 💥 - as it can't fit all of the cases, but themes like even https://startbootstrap.com/ are proving it's possible without a massive amount of work to be done.

What you think?

**Reply by TimGeyssens** [reply]
ok yeah multi-purpose, need to look into that...look pretty sweet... was just thinking real estate since that can show some nice stuff like advanged search (with location)...filtering...

**Reply by CarlSargunar** [reply]
I would also lean on a real world use case, but not one for a specific sector - humans don't tend to have the greatest imagination sometimes

I was thinking - would it be worth putting the site together to actually discuss host info about the starter kit itself? Like a mini-brochureware site for the community starter kit built on the starter kit, with associated app etc. 

It would be the most meta umbraco project yet

**Reply by CarlSargunar** [reply]
The other option I was thinking of was a not-for-profit based example site.  Non-profits don't have the best budgets at the best of times, so anything that can reduce the amount of work required may be a good thing to do

**Reply by TimGeyssens** [reply]
yeah love the non profit idea,what would be a good case?

**Reply by skttl** [reply]
You could find a non-profit template on themeforest like this https://themeforest.net/category/site-templates/nonprofit/charity and implement all the functions in an Umbraco website. Not sure if the license allows it, but then you could also just take inspiration from it.

**Reply by TimGeyssens** [reply]
thanks @skttl any ideas on a search/filter option on those? Since most of them are just content I suppose..maybe events?

**Comment by melvinvanrookhuizen** [comment]
contirbution wise, how would this work for non-dev? Me and my partner have built an all-in solution for Umbraca that beats UNO by a mile. I amon 'the other side' of the websites. where the people live ;-) and happy to share input on concept, checking designs, UIX and functionality (that is what I do) but no hands-on coding. So that would require different tooling outside Github I guess

**Reply by TimGeyssens** [reply]
sound good! what tooling do you suggest?

**Comment by melvinvanrookhuizen** [comment]
On the real-life usecase note... A real project might even be better. Open up a 'request channel' for non-profits. So the community does actually 'give away' sites. I would be happy to chip in with a 'donation/sponsor module' we ar currently converting. I would be happy to spend time on the good cause. Ist better to work towards areal goal, and its brilliant marketing for Umbraco.

**Reply by TimGeyssens** [reply]
nice! Sound perfect, yes good idea!

**Reply by TimGeyssens** [reply]
@melvinvanrookhuizen mind telling some details about the module... so the designers know which frontend bits it has... #23 , thanks!

**Reply by melvinvanrookhuizen** [reply]
@TimGeyssens my lead developer is not available this week due to some personal circumstances. When he is back next week, I will see what details I can send ok? We have some V7 demo somwehere that might help. V8 by eo this year.

**Reply by TimGeyssens** [reply]
sure no worries, yeah if we just know how it works frontend wise it is ok, that way we can design it.
